hooks/useConversation.ts

```diff
@@ -51,24 +52,53 @@ const useConversation = (
       }
       setLoading(false)
     }
+    listMessages()
+  }, [conversation, convoMessages])
+
+  useEffect(() => {
+    if (!conversation) return
     const streamMessages = async () => {
-      const stream = await conversation.streamMessages()
-      setStream(stream)
+      stream = await conversation.streamMessages()
       for await (const msg of stream) {
         if (dispatchMessages) {
           await dispatchMessages({
             peerAddress: conversation.peerAddress,
             messages: [msg],
           })
         }
+        if (
+          latestMsgId !== msg.id &&
+          Notification.permission === 'granted' &&
+          msg.senderAddress !== walletAddress &&
+          !browserVisible
+        ) {
+          new Notification('New Message On XMTP', {
+            body: `From ${msg.senderAddress}`,
```

Comment:
Let's display the ENS here if one is available, and otherwise fall back to display shortAddress 

https://github.com/xmtp/example-chat-react/blob/ee66ff59445fd86c37c488031a8646a3bd4973bb/components/Address.tsx#L9

Following Slack's convention this would look like:

> **saulmc.eth**
> this is a message

> **0x5992...85F1**
> this is a message

<img width="352" alt="CleanShot 2022-09-16 at 21 41 58@2x" src="https://user-images.githubusercontent.com/510695/190840788-884fa045-a472-42f1-82cf-df76a432795e.png">